lib/modules/manager/kotlin-script/index.ts

```diff
@@ -0,0 +1,14 @@
+export { extractPackageFile } from './extract';
+
+import { ProgrammingLanguage } from '../../../constants';
+import { MavenDatasource } from '../../datasource/maven';
+import * as gradleVersioning from '../../versioning/gradle';
+
+export const language = ProgrammingLanguage.Kotlin;
+
+export const defaultConfig = {
+  fileMatch: ['^.*\\.main\\.kts$'],
```

Comment:
if you want to support all `kts` except `gradles` use this `^(?!.*gradle).*\\.kts$`

Reply:
hidden comment

Reply:
We decided on another approach - ignoring Gradle files in extraction logic.